docs/Build/Linux.md

```diff
@@ -134,3 +134,36 @@ Copyright (c) 2008-2020 Juan-Pablo Caceres, Chris Chafe.
 SoundWIRE group at CCRMA, Stanford University
 ```
 
+### Running Two Versions of JackTrip in Parallel
+One level above the project directory of your current JackTrip installation, make another folder, enter it and clone the JackTrip repository again:
+
+```sh
+$ mkdir jacktrip-1.x.x
+$ cd jacktrip-1.x.x
+$ git clone --recurse-submodules https://github.com/jacktrip/jacktrip.git
```

Comment:
If you add jacktrip-1.x.x at the end here, you don't have to create the directory beforehand.